components/tvshows/TVEpisodes.bs

```diff
@@ -59,11 +58,11 @@ end sub
 
 ' get the currently focused item
 function getFocusedItem() as dynamic
-    if not isValid(m.top.focusedChild) or not isValid(m.top.focusedChild.focusedChild)
+    if not isValid(m.top.focusedChild)
```

Comment:
We can combine and clean up this valid check and the below focusedChild.content valid check.

```suggestion
        if not isChainValid(m.top.focusedChild, "content") then return invalid
```